back/gemfile-que-1/Gemfile

```
@@ -0,0 +1,236 @@
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'annotate'
+  gem 'redcarpet'
+  gem 'spring'
+  gem 'spring-commands-rspec'
+  gem 'spring-watcher-listen', '~> 2.1'
+end
+
+group :test do
+  gem 'rubyXL'
+  gem 'shoulda-matchers', '~> 5.3.0'
+  gem 'webmock', '~> 3.18'
+
+  # Adds rspec matchers and helpers for testing html structure and styling.
+  gem 'capybara', '~> 3.39'
+  # gem 'bullet'
+end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+# gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'active_model_serializers', '~> 0.10.12'
+gem 'pundit', '~> 2.3.0'
+
+gem 'jwt', '~> 2.7.0'
+gem 'que', '~> 1.4.1'
+gem 'que-web', '~> 0.10.0'
+
+gem 'activerecord-import', '~> 1.4'
+gem 'activerecord-postgis-adapter', '~> 8.0'
+gem 'json-schema', '~> 4.0'
+
+gem 'carrierwave', '~> 3.0.5'
+gem 'carrierwave-base64', '~> 2.10'
+gem 'fog-aws', '~> 3.19'
+
+gem 'api-pagination', '~> 5.0.0'
+gem 'kaminari', '~> 1.2'
+gem 'rails-i18n', '~> 7.0.7'
+
+gem 'awesome_nested_set', '~> 3.5.0'
+gem 'axlsx', '3.0.0.pre'
+gem 'counter_culture', '~> 3.3'
+gem 'groupdate', '~> 4.1'
+gem 'icalendar', '~> 2.9'
+gem 'interactor'
+gem 'interactor-rails'
+gem 'liquid', '~> 5.4'
+gem 'mini_magick', '~> 4.12'
+gem 'nokogiri', '~> 1.14.3'
+gem 'pg_search', '~> 2.3.5'
+gem 'premailer-rails', '~> 1.12.0'
+gem 'rest-client'
+gem 'rgeo-geojson'
+gem 'rubyzip', '~> 1.3.0'
+gem 'stackprof', '~> 0.2.25'
+
+gem 'acts_as_list', '~> 1.1'
+gem 'bunny', '>= 2.7.2'
+gem 'faker'
+gem 'faraday-jwt', '~> 0.1.0'
+gem 'ice_cube', '~> 0.16'
+gem 'okcomputer'
+gem 'omniauth', '~> 1.9.1'
+gem 'omniauth-facebook'
+gem 'omniauth-google-oauth2'
+gem 'omniauth_openid_connect', '~> 0.7.1'
+gem 'omniauth-rails_csrf_protection'
+gem 'omniauth-saml', '~> 1.10.3'
+gem 'order_as_specified'
+gem 'prawn', '~> 2.4'
+gem 'prawn-grouping', '~> 0.2.0'
+gem 'scenic'
+gem 'sentry-rails'
+gem 'sentry-ruby'
+gem 'simple_segment', '~>1.5'
+
+# Also required here to be able to initialize Mailgun in
+# e.g. production.rb, which would otherwise result in an
+# "undefined method 'mailgun_settings=' for ActionMailer::Base:Class"
+# exception.
+gem 'aws-sdk-bedrockruntime', '~> 1'
+gem 'aws-sdk-s3', '~> 1'
+gem 'bootsnap', '~> 1', require: false
+gem 'dalli', '~> 3.2.4'
+gem 'jsonapi-serializer'
+gem 'mailgun-ruby', '~>1.2.8'
+gem 'rack-attack', '~> 6'
+gem 'rails_semantic_logger'
+gem 'rinku', '~> 2'
+
+gem 'intercom', '~> 4.2'
+# mjml-rails cannot find the MJML parser when installed
+# through the emails engine and is therefore specified
+# in the main app.
+gem 'cloudfront-rails', '~> 0.4'
+gem 'combine_pdf'
+gem 'google-cloud-document_ai'
+gem 'mjml-rails', '~> 4.9'
+
+free_engines = %w[
+  document_annotation
+  email_campaigns
+  frontend
+  onboarding
+  polls
+  seo
+  surveys
+  volunteering
+]
+
+free_engines.each do |engine_name|
+  path = "../engines/free/#{engine_name}"
```

The only difference with main Gemfile is here:
```suggestion
  path = "engines/free/#{engine_name}"
```